packages/@ourworldindata/types/src/OwidVariable.ts

```diff
@@ -107,6 +108,7 @@ export interface OwidVariableDimension {
 export interface OwidVariableDimensions {
     years: OwidVariableDimension
     entities: OwidVariableDimension
+    values?: OwidVariableDimension
```

Comment:
Minor detail, but: I don't really like the `values?.values` that's going on here.
I.e. two levels that are both names `values`.

Reply:
yeah, I don't like it either, but considering that [data json files](https://api-staging.owid.io/staging-site-ordinal-types/v1/indicators/830988.data.json) use `years`/`entities`/`values` as top-level keys, it seems the most consistent?